whylog/config/rule.py

```diff
@@ -19,10 +19,10 @@ def serialize(self):
             "constraints": self._constraints,
         }
 
-    def get_new_parsers(self, old_parsers):
+    def get_new_parsers(self, parser_name_generator):
         new_parsers = []
         for parser in itertools.chain([self._effect], self._causes):
```

Comment:
This collection will have only a few elements. I suppose it will be about 5 elements on average in this list, maybe 20 maximally. So why it is needed?